packages/Webkul/Shop/src/Resources/views/checkout/cart/mini-cart.blade.php

```diff
@@ -176,7 +176,7 @@ class="m-0 ml-[0px] block mx-auto bg-navyBlue text-white text-base w-full font-m
                  * Action.
                  */
                 this.$emitter.on('test-cart', (cart) => {
```

Comment:
Event name should be appropriate

Reply:
Also, this event callback should accept cart not cart.cart

Reply:
Sir fixed